app/javascript/modCenter/singleArticle/__tests__/singleArticle.test.jsx

```diff
@@ -197,8 +197,9 @@ describe('<SingleArticle />', () => {
       </Fragment>,
     );
 
-    const button = getByTestId(`mod-article-${article.id}`);
-    button.click();
+    const detailsElement = getByTestId(`mod-article-${article.id}`);
+    const summarySection = detailsElement.getElementsByTagName("summary")[0];
+    summarySection.click();
```

Comment:
This is causing all the specs to fail. Clicking on the summary section is toggling the `open` attribute value but isn't firing the onToggle event handler :/